src/chai-form.ts

```diff
@@ -135,6 +145,10 @@ export class ChaiForm extends LitElement {
        * Inputs and the button have their own border styles.
        */
       --chai-form-border: 1px solid #ccc;
```

Comment:
These look like they will conflict with the `--chai-form-border` composite property. A cleaner solution would be to set the `--chai-form-border` property using string concatenation of your border style/width/color properties, so the public API of `chai-form` for styles doesn't have to change.